Formula/smithy-language-server.rb

```diff
@@ -0,0 +1,18 @@
+class SmithyLanguageServer < Formula
+  desc "Language Server Protocol implementation for the Smithy IDL"
+  homepage "https://github.com/awslabs/smithy-language-server"
+  url "https://github.com/awslabs/smithy-language-server"
+  version "0.2.2"
+  sha256 :no_check
```

Comment:
can you check out the other gradle formulae on the formula setup? Thanks!

Reply:
Hi @chenrui333, thanks for the feedback. I'm not sure what the other formula is I'm sorry. Could you please send me a link so that I can understand a little better? 😄

Reply:
@chenrui333 I'm having a little trouble with the `sha256` field. I'm trying to download from `https://github.com/awslabs/smithy-language-server` where they don't have releases yet. The `:no_check` is failing, with the output expecting a SHA. Is it possible to skip this? I'd read in the cookback that `:no_check` is supposed to skip it but I don't see any other formulas using this and the SHA changes each time I run the build from source command to test it. Do you have any ideas on how to resolve this? It seems like it would be a common issue

Reply:
Your URL just downloads the webpage. If they don't have releases we can't ship this. How would users know what they install?

Reply:
Thanks @SMillerDev, I understand now why I'm getting errors. I had mistakenly thought that it cloned the repository. I've submitted [a PR](https://github.com/awslabs/smithy-language-server) to their repository to publish releases, so will revisit when done.

In the mean time, could I ask about the stars requirement? If the repository is maintained by AWS, is it possible to get an exception to the popularity requirement? Or do we need to wait a little longer for it to reach more stars?